.azure-pipelines/ultimate-pipeline.yml

```
@@ -373,6 +373,10 @@ stages:
       displayName: Upload linux-x64 monitoring home
       artifact: linux-monitoring-home-$(baseImage)
 
+    - publish: $(System.DefaultWorkingDirectory)/shared/symbols
+      displayName: Upload linux-x64 symbols
+      artifact: linux-symbols-home-$(baseImage)
```

nit: maybe a better name, if we add this to the release?
```suggestion
      artifact: linux-native-symbols-$(baseImage)
```